src/state/tokenPrices/hooks.ts

```diff
@@ -68,6 +68,8 @@ const useTokenPricesLocal = (
 
           dispatch(updatePrices(formattedPrices))
         }
+      } catch (e) {
+        // empty
```

Comment:
Is this intended empty? Because this is useless

Reply:
There's no catch before. So I just added an empty catch :V